src/main.rs

```diff
@@ -106,7 +106,6 @@ pub fn create_lock_timeout(
 			que.lock.update(|item| item.retain(|ids| *ids != id));
 
 			que.tooltip.set(Vec::new()); // reset all tooltips before locking
-			db_timeout.clear_hash();
```

Comment:
This should be replaced with `lock()` no? Or the timeout won't lock properly

Reply:
The next line changes the state and when we load that state we call the db.lock().

Reply:
"let's keep it in the create_lock_timeout function and remove it from the match of the dyn container". - Dominik